backend/courses/models.py

```diff
@@ -1419,3 +1419,40 @@ def create_or_update_user_profile(sender, instance, created, **kwargs):
     if created and instance.email != "":
         instance.profile.email = instance.email
     instance.profile.save()
+
+
+
+class Friendship(models.Model):
+    """
+    Used to track friendships along with requests status 
+    """
+    
+    requestSender = models.ForeignKey(
```

Comment:
snake_case is typically considered to be more idiomatic for Python (according to PEP 8): https://peps.python.org/pep-0008/#function-and-variable-names